src/caretogether-pwa/src/Families/FamilyScreen.tsx

```diff
@@ -386,6 +402,23 @@ export function FamilyScreen() {
               </Grid>}
           </Grid>
           <Grid container spacing={0}>
+			<Grid item xs={12}>
+              <h3 style={{ marginBottom: 0 }}>Communities</h3>
+				{familyCommunityInfo?.map(communityInfo => {
+					return (
+						<Card variant="outlined" sx={{ paddingBottom: '1rem' }}>
```

Comment:
Can you add `cursor: pointer` to this as an affordance for desktop users that this is clickable?

Reply:
Cursor updated